verkko/2.0/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM mambaorg/micromamba:1.5.6 as app
+
+ARG VERKKO_VER="2.0"
+
+USER root
+
+WORKDIR /
+
+LABEL base.image="mambaorg/micromamba:1.5.6"
+LABEL dockerfile.version="1"
+LABEL software="Verkko"
+LABEL software.version="${VERKKO_VER}"
+LABEL description="Verkko is a hybrid genome assembly pipeline developed for telomere-to-telomere assembly of PacBio HiFi or Oxford Nanopore Duplex and Oxford Nanopore simplex reads."
+LABEL website="https://github.com/marbl/verkko"
+LABEL license="https://github.com/marbl/verkko/blob/master/README.licenses"
+LABEL maintainer="Kutluhan Incekara"
+LABEL maintainer.email="kutluhan.incekara@ct.gov"
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    procps \
+    curl && \
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+RUN micromamba install --name base -c conda-forge -c bioconda verkko=${VERKKO_VER} && \
+    micromamba clean -a -y
+
+ENV PATH="/opt/conda/bin/:${PATH}" \
+    LC_ALL=C.UTF-8
+
+CMD [ "verkko", "--help" ]
+
+WORKDIR /data
+
+## Test ##
+FROM app as test
+
+ENV ENV_NAME="base"
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+
+RUN apt-get update && apt-get install --no-install-recommends -y curl
+
+# test data
+RUN curl -L https://obj.umiacs.umd.edu/sergek/shared/ecoli_hifi_subset24x.fastq.gz -o hifi.fastq.gz &&\
+    curl -L https://obj.umiacs.umd.edu/sergek/shared/ecoli_ont_subset50x.fastq.gz -o ont.fastq.gz
+
+# verkko test run
+RUN verkko -d asm --hifi ./hifi.fastq.gz --nano ./ont.fastq.gz &&\
+    head asm/assembly.hifi-coverage.csv &&\
+    head asm/assembly.fasta
```

Comment:
This prints a lot to the screen. I have a hunch that `head asm/assembly.fasta` might the culprit. Can we try something like `wc -l asm/assembly.fasta`?

```
#11 225.4     resources: tmpdir=/tmp
#11 225.4 
#11 225.4 Reading graph from stdin
#11 225.4 [Tue Feb 27 17:49:59 2024]
#11 225.4 Finished job 0.
#11 225.4 64 of 64 steps (100%) done
#11 225.4 Complete log: .snakemake/log/2024-02-27T174614.567918.snakemake.log
#11 225.5 node	coverage	length
#11 225.5 utig4-0	23.78	3421198
#11 225.5 utig4-1	7.67	4526
#11 225.5 utig4-2	8.00	4518
#11 225.5 >contig-0000001
#11 225.5 
```

Reply:
I think contig sequences are written in one line, so head gives a lot of outbut. I limit output with 1000 bytes. That should work.